src/main/java/com/titarenko/config/SpringSecurityConfiguration.java

```diff
@@ -0,0 +1,28 @@
+package com.titarenko.config;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.authentication.builders.AuthenticationManagerBuilder;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+
+@Configuration
+@EnableWebSecurity
+public class SpringSecurityConfiguration extends WebSecurityConfigurerAdapter {
+
+    @Override
+    protected void configure(HttpSecurity http) throws Exception {
+        http.csrf().disable().authorizeRequests()
+                .antMatchers("/**").hasAnyRole("ADMIN")
+                .anyRequest().authenticated()
+                .and().formLogin().defaultSuccessUrl("/employee/", true)
```

Comment:
Could you change it to `defaultSuccessUrl("/", true)` and map both path `/employee` and `/` to the employee page, please